Cargo.toml

```diff
@@ -22,6 +22,7 @@ is-it-maintained-open-issues = { repository = "enarx/sgx" }
 
 [features]
 rcrypto = ["rand", "rsa", "sha2", "num-integer", "num-traits"]
+ioctls = ["iocuddle", "libc"]
```

Comment:
This isn't correct since this will pull in `libc` as a runtime dependency. But `libc` isn't a runtime dependency.

Reply:
Just remove this line altogether.